book/src/terminology.md

```diff
@@ -213,6 +213,10 @@ The public key of a [keypair](#keypair).
 Storage mining client, stores some part of the ledger enumerated in blocks and
 submits storage proofs to the chain. Not a full-node.
 
+#### root
```

Comment:
How about "root block"?

Reply:
I'm resistant.

We don't log or say `root block` anywhere, and there are no conflicts with any other `root Xs` yet.

Reply:
How about "root: see *rooted block*"

Reply:
Or "the highest *rooted block*

Reply:
hm.  We say "a root", which is a useful thing in the context of a running validator.

still resistant ;)